community/rfcs/24-10-02-GenAIExamples-001-Image_and_Audio_Support_in_MultimodalQnA.md

```diff
@@ -0,0 +1,254 @@
+# 24-10-02-GenAIExamples-001-Image_and_Audio_Support_in_MultimodalQnA
+
+## Author(s)
+
+[Melanie Buehler](https://github.com/mhbuehler), [Mustafa Cetin](https://github.com/MSCetin37), [Dina Jones](https://github.com/dmsuehir), [Omar Khleif](https://github.com/okhleif-IL)
+
+## Status
+
+Under review
+
+## Objective
+
+The [MultimodalQnA](https://github.com/opea-project/GenAIExamples/tree/main/MultimodalQnA) megaservice in
+[GenAIExamples](https://github.com/opea-project/GenAIExamples) currently supports text queries with a response based on
+the context derived from a collection of videos. This RFC expands upon that and proposes the addition of images, images
+with text, and audio data types for both the ingested data and the user query.
+
+## Motivation
+
+As the [Multimodal RAG RFC](https://github.com/opea-project/docs/blob/01597aabeaf4c5d171bdc8cd9f7bccdd9e64f697/community/rfcs/MM-RAG-RFG.md)
```

Comment:
Once this PR is merged, we can update this file and point it to where it resides on https://github.com/opea-project/docs

Reply:
I agree. I'll keep an eye on it.